rdr_service/offline/main.py

```diff
@@ -906,6 +907,37 @@ def nph_biobank_inventory_file_import():
     return '{"success": "true"}'
 
 
+@app_util.auth_required_cron
+def ptsc_test_participant_cleanup_request():
+    date = datetime.today()
+    if date.month == 12:
+        month = 1
+    else:
+        month = date.month + 1
```

Comment:
Importing and using `relativedelta` from the `dateutil.relativedelta` module could be helpful here. `date = datetime.today() + relativedelta(months=1)` would be a little more concise, and would increment the year when needed too.